virttest/utils_test/__init__.py

```diff
@@ -698,6 +698,27 @@ def run_autotest(vm, session, control_path, timeout,
     from autotest.client.shared.settings import settings
     section_values = settings.get_section_values
 
+    def install_client_tests(url, autotest_path, timeout=600):
```

Comment:
Basically you can use `session` to do the same inside the `extract` function https://github.com/avocado-framework/avocado-vt/pull/394/commits/634a2c029f1b9868b00fd55aa38be50c7b900bdf#diff-4bcc7565386c108b748a19044cade2e3R795

this way we'd avoid the necessity to run this as root.